DevelopedCalculator.js

```diff
@@ -1,6 +1,9 @@
 class DevelopedCalculator extends Calculator {
     render() {
         super.render()
+        const history = document.createElement('div');
+        const historyDiv_ = document.createTextNode("History:");
+        const historyDiv = this.hostElm.prepend(historyDiv)
```

Comment:
how to add a text content to a HTM element?

Also when you prepent the historyDiv to the `hostElm` you don't need to assign that to a variable